demos/python_demos/object_detection_retinaface_demo/README.md

```diff
@@ -0,0 +1,56 @@
+# Object Detection RetinaFace Demo
+
+This demo showcases Face Detection with RetinaFace. The task is to identify faces as axis-aligned boxes
+and their keypoints (facial landmarks) in an image.
+
+## How It Works
+
+On the start-up, the application reads command-line parameters and loads a network to the Inference
+Engine. Upon getting a frame from the OpenCV VideoCapture, it performs inference and displays the results.
+
+## Running
+
+Running the application with the <code>-h</code> option yields the following usage message:
+
+```
+usage: object_detection_retinaface_demo.py [-h] -m MODEL
+                                           [-i INPUT [INPUT ...]] [-d DEVICE]
+                                           [-f_pt FACE_PROB_THRESHOLD]
+                                           [-m_pt MASK_PROB_THRESHOLD]
+                                           [--no_show]
+                                           [-u UTILIZATION_MONITORS]
+
+Options:
+  -h, --help            Show this help message and exit.
+  -m MODEL, --model MODEL
+                        Required. Path to an .xml file with a trained model.
+  -i INPUT [INPUT ...], --input INPUT [INPUT ...]
+                        path to video or image/images
+  -d DEVICE, --device DEVICE
+                        Optional. Specify the target device to infer on; CPU,
+                        GPU, FPGA, HDDL or MYRIAD is acceptable. The demo will
+                        look for a suitable plugin for device specified.
+                        Default value is CPU
+  -f_pt FACE_PROB_THRESHOLD, --face_prob_threshold FACE_PROB_THRESHOLD
+                        Optional. Probability threshold for face detections
+                        filtering
+  -m_pt MASK_PROB_THRESHOLD, --mask_prob_threshold MASK_PROB_THRESHOLD
```

Comment:
Please, rename `-m_pt` key, because `-m_` suffix uses for settting models (e.g. `-pt_m`, `-pt_f`).

Reply:
Don't forget update Readme